dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/compilation/VisitorBase.java

```diff
@@ -29,25 +29,26 @@
 
 class VisitorBase implements EventVisitor<List<Event>> {
 
-    protected boolean forceStart;
```

Comment:
How is force start handled now?

Reply:
It is in `ThreadCreation` as that pass is responsible for setting up all thread-related code.